styles.css

```diff
@@ -0,0 +1,39 @@
+.calculator {
+  height: 230 px;
+  width: 202px;
+  background-color: rgb(49, 48, 48);
+  box-sizing: border-box;
+  position: fixed;
+  display: flex;
+  flex-wrap: wrap;
+  border: 1px solid black;
+  border-radius: 3px;
+}
+
+.result {
+  background-color: rgb(235, 230, 224);
+  border: 1px solid black;
+  width: 200px;
+  height: 50px;
+  font-family: sans-serif;
+  font-size: 30px;
+  align-items: center;
+  border-radius: 3px;
+}
+
+button{
```

Comment:
Never ever style the HTML elements directly, this is bad pratice.

Reply:
done